config/config/wc-config.yaml

```
@@ -70,6 +70,64 @@ opa:
     tags:
      - latest
 
+## Hierarchical namespace controller configuration, enable in common-config.yaml
+hnc:
+  ## Enables HA mode for hnc webhooks
+  ha: true
+
+  ## Included namespaces
+  includedNamespacesRegex: ""
```

You are using `includedNamespaceRegex` in the template
```suggestion
  includedNamespaceRegex: ""
```